.env.test

```diff
@@ -0,0 +1,2 @@
+REFERENCE_ID=gpdefvsxamnscceccczu
```

Comment:
We should not check in the values here - just the keys - this is a security issue.

Reply:
Ok 
I did it without checking, I'll be careful in the next ones